### DISCUSSION TITLE
Bethesda Blender Exporter

### DISCUSSION THREAD

**Discussion by robertgk2017** [BadDogSkyrim/PyNifly#249] [Ideas]
Beth released a Blender Exporter.   However it doesnt allow importing existing .nifs.

It could be a very useful source of RE knowledge on the format and adding/fixing stuff for pyNifly.  def recommend screwing around with it.

**Comment by BadDogSkyrim** [comment]
Sounds cool. Do you have a link? The one I found just points me at the SSE CK on Steam, and the date on it hasn't been updated.

**Reply by robertgk2017** [reply]
Its shipped with the CK itself.  Will be under [Game Folder]\Tools
